networkx/algorithms/centrality/flow_matrix.py

```diff
@@ -16,9 +16,9 @@ def flow_matrix_row(G, weight='weight', dtype=float, solver='lu'):
     C = solvername[solver](L, dtype=dtype) # initialize solver
     w = C.w # w is the Laplacian matrix width
     # row-by-row flow matrix
-    for u,v,d in G.edges(data=True):
+    for u,v in sorted(sorted((u,v)) for u,v in G.edges()):
```

Comment:
I probably shouldn't have hidden this bug fix in a larger PR.
But I don't mind leaving it here.